install.py

```diff
@@ -222,10 +238,7 @@ def config_account(self):
         print('* Username and password set successfully!')
 
     def detect_ip(self):
-        s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
-        s.connect(('www.baidu.com', 80))
-        ip = s.getsockname()[0]
-        s.close()
+        ip = urllib2.urlopen('http://ip.42.pl/raw').readline()
```

Comment:
This is a stable and free external resources

Reply:
urModule urllib2, please compatible with python2 and python3